docs/cookbook/disabling-localised-urls.rst

```diff
@@ -0,0 +1,52 @@
+How to disable localised URLs?
+==============================
+
+**1.** Customise application routing in ``app/config/routing.yml``.
+
+Replace:
+
+.. code-block:: yaml
+
```

Comment:
I'm confused in what cases should I include comment with filename. 

When I proposed that a week ago in a doc PR I heard it wasn't needed because there was namespace declaration. In this case the step title mentions the filename, so it looks redundant here.